### PR TITLE
Created Verify Window

### PR DESCRIPTION
**Pull request by Divnoor-C**
Created the verify window, with packages in use_case and interface_adapter

### CODE REVIEW

**Comment by shahkabir1** [comment]
Looks good

File: src/main/java/use_case/verify/VerifyOutputBoundary.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+    void prepareSuccessView(VerifyOutputData outputData);
+    /**
+     * Prepares the failure view for the Verify Email Use Case.
+     * @param errorMessage the explanation of the failure
+     */
+
+    void prepareFailView(String errorMessage);
+
+    void switchToSignUpView();
```
Comment (Mahan-H) [inline src/main/java/use_case/verify/VerifyOutputBoundary.java:21]: add missing javadoc

File: src/main/java/use_case/verify/VerifyInputData.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+ */
+
+public class VerifyInputData {
+    private final String verifyCode;
+
+    public VerifyInputData(String verifyCode) {
+        this.verifyCode = verifyCode;
+    }
+
+    public String getVerifyCode() {
```
Comment (Mahan-H) [inline src/main/java/use_case/verify/VerifyInputData.java:14]: add missing javadoc

File: src/main/java/use_case/verify/VerifyInputBoundary.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+ */
+public interface VerifyInputBoundary {
+    /**
+     * Execute the Verify Email Use Case.
+     * @param verifyInputData the input data for this use case
+     */
+
+    void execute(VerifyInputData verifyInputData);
+
+    void switchToSignUpView();
```
Comment (Mahan-H) [inline src/main/java/use_case/verify/VerifyInputBoundary.java:14]: add missing javadoc

**Pull request merged**